psutil/_psosx.py

```diff
@@ -212,6 +213,84 @@ def disk_partitions(all=False):
 # =====================================================================
 
 
+def get_smc_temperatures(smc_keys, iter_num, iteratable):
+    """ Receives a dictionary of SMC keys, and a number possible keys
+    If keys can be iterated, replaces the expected %d in the key.
+    Returns a list of all smc_get_temperature results higher than 0
+    """
+    ret = list()
+    if iteratable:
+        for idx in range(iter_num):
+            for key, label in smc_keys.items():
+                result = cext.smc_get_temperatrue(key % idx)
+                if int(result) <= 0:
```

Comment:
`int` is unnecessary. Also, I suggest to round the result to the first decimal digit as in `result = round(result, 1)`